packages/lesswrong/components/tagging/PostsItemTagRelevance.tsx

```diff
@@ -21,13 +22,21 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
   vertLayoutVoteUp: {
     position: "absolute",
-    left: isFriendlyUI ? 9 : 8,
-    top: -15,
+    left: isFriendlyUI ? 10.5 : 10,
+    top: forumSelect({
+      LessWrong: -17,
+      AlignmentForum: -15,
+      default: -16
+    })
   },
   vertLayoutVoteDown: {
     position: "absolute",
-    left: isFriendlyUI ? 9 : 8,
-    top: isFriendlyUI ? 10 : 9,
+    left: isFriendlyUI ? 10.5 : 10,
+    top: forumSelect({
+      LessWrong: 8,
+      AlignmentForum: 10,
+      default: 10
+    })
```

Comment:
Ah, because my testing failed. We don't have tag relevance on the right (we replace karma on the left) and I thought I was adjusting the buttons on the left. Fixed now.